internal/infrastructure/middleware/metrics_test.go

```diff
@@ -93,4 +93,10 @@ func TestMetricsMiddlewareWithConfigCreation(t *testing.T) {
 
 	assert.Equal(t, http.StatusOK, resp.Code)
 	assert.Equal(t, "Ok", resp.Body.String())
+
+	// Check skipper
+	path = "/ping"
+	resp = httptest.NewRecorder()
+	req = httptest.NewRequest(http.MethodGet, path, nil)
+	e.ServeHTTP(resp, req)
```

Comment:
Is the test complete? I'm not sure how it is "checking the skipper".

Reply:
thanks! I have updated the test to include assertion for the Skipper call.